implementation/src/main/java/io/smallrye/metrics/setup/MetricCdiInjectionExtension.java

```diff
@@ -97,7 +103,8 @@ public MetricCdiInjectionExtension() {
     }
 
     private void addInterceptorBindings(@Observes BeforeBeanDiscovery bbd, BeanManager manager) {
-        log.info("MicroProfile: Metrics activated");
+        log.info("MicroProfile: Metrics activated (SmallRye Metrics 2.0 version: "
```

Comment:
Maybe make it a debug or trace?

Not sure we need to show the version at all times

Reply:
Hm I'd say I personally prefer INFO. The log will have a doubled value now - saying that metrics are active, and the impl version.
Also looking at the boot logs of WildFly 16, I see 10 similar INFO logs containing versions of different components, so I guess this must be quite "normal" :)

Reply:
Possibly, but from the Quarkus perspective we want less logs to improve startup.

Maybe we need a way to configure logging by environment?

Reply:
This code doesn't apply to Quarkus anyway, because this is within the CDI extension and Quarkus uses different mechanisms because it does not support CDI extensions, so this class is not used at all.

Reply:
I still don't think info logging is appropriate just to say something started

Reply:
I personally like it, but I don't care much :-)

Reply:
I'll merge it, if we get any complaints we can always get rid of it